plugins/modules/zos_operator_action_query.py

```diff
@@ -387,11 +413,26 @@ def __init__(self, message):
         )
 
 
-class OperatorCmdError(Error):
-    def __init__(self, message):
-        self.msg = 'An error occurred during issue the operator command, the response is "{0}"'.format(
-            message
-        )
+class OperatorQueryResult:
+    def __init__(
+        self,
+        rc,
+        stdout,
+        stderr
+    ):
+        """Response object class to manage the result from executing a command
+        to query for actionable messages. Class will also generate a message
+        by concatinating stdout and stderr
+
+        Arguments:
+            rc {str} -- The return code
+            stdout {str} -- The standard out of the command run
+            stderr {str} -- The standard error of the command run
+        """
+        self.rc = rc
+        self.stdout = stdout
+        self.stderr = stderr
+        self.message = stdout + " " + stderr
 
```

Comment:
Style question: If you have out and err on the same return, should the error section, if populated, be preceded by something like 'error'?  I'm thinking the output could be out=OK or out=None, space, then error might not be clear.

Reply:
@richp405 - you bring up a good point, I was following what they had in place, some of the difficultly here resides in the fact I can't get a non zero RC from the `opercmd`. Before my edits, I think what the developer did was use both the message field for a way to share with the user a failure and success thinking only one would populate when the two were concatenated, but with the new changes we don't send back the `message` field so I think its safe just return stdout for `message` such that `self.message = stdout + " " + stderr` is updated to `self.message = stdout`.  

Given the `message` field only is sent back in the success case and only stdout interest users for operator actions, i see no reason to send back stderr. 

Good catch, ill update.